File: Day01_Music/app/src/main/java/day01/nguyenhongphat0/day01_music/MainActivity.java
package day01.nguyenhongphat0.day01_music;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button play, stop;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        play = findViewById(R.id.btnPlay);
        stop = findViewById(R.id.btnStop);
        play.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, PlayMusicService.class);
                startService(intent);
            }
        });
        stop.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, PlayMusicService.class);
//                stopService(intent);
            }
        });
    }
}

File: kotlin/Day01/app/src/main/java/day01/nguyenhongphat0/day01/MainActivity.kt
package day01.nguyenhongphat0.day01

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val playButton = findViewById<Button>(R.id.playButton)
        val stopButton = findViewById<Button>(R.id.stopButton)
        val intent = Intent(this, PlayMusicService::class.java)
        playButton.setOnClickListener { startService(intent) }
        stopButton.setOnClickListener { stopService(intent) }
    }
}
